modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/ColorPickerInput.es.js

```diff
@@ -15,12 +15,18 @@
 import ClayColorPicker from '@clayui/color-picker';
 import React, {useState} from 'react';
 
+function normalizeColor(colorValue) {
+	return colorValue.replace('#', '');
+}
+
 const HEX_COLOR_REGEX = /^#?[0-9A-F]{3}(?:[0-9A-F]{3})?$/i;
 
 const ColorPicker = ({color, label, name}) => {
 	const [colorValue, setColorValue] = useState(color);
 	const [customColors, setCustomColors] = useState([]);
 
+	const normalizeColorValue = normalizeColor(colorValue);
```

Comment:
Not a blocker, but there really isn't a need to extract `normalizeColor` into its own function. You could simply just inline the `.replace`. And I would also rename the variable to be a bit more explicit

```suggestion
	const noHashColorValue = colorValue.replace('#', '');
```

Reply:
Truly appreciate the feedback! Made the changes.